program-runtime/src/invoke_context.rs

```diff
@@ -152,7 +152,7 @@ pub struct EnvironmentConfig<'a> {
     pub blockhash: Hash,
     pub blockhash_lamports_per_signature: u64,
     epoch_total_stake: Option<u64>,
-    epoch_vote_accounts: Option<&'a VoteAccountsHashMap>,
+    epoch_vote_stake: &'a HashMap<Pubkey, u64>,
```

Comment:
Can we unify the APIs for total stake and vote account stake here? I'm fine with either of these two options:
- Make `epoch_total_stake` just a `u64`, defaulting to zero everywhere.
- Make both fields an `Option`.

Maybe the first is cleaner?